source/core-developers/fetch-metadata-interceptor.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Fetch Metadata Interceptor
+parent:
+    title: Interceptors
+    url: interceptors.html
+---
+
+# Fetch Metadata Interceptor
+
+## Description
+
+An interceptor that implements Fetch Metadata on incoming requests used to protect against CSRF, XSSI, and cross-origin information leaks. Uses a default Resource Isolation Policy to programmatically reject cross-origin requests.
+
+A Resource Isolation Policy is a strong defense in-depth mechanism that prevents the resources on a server from being requested by external websites. This policy can be enabled either for all endpoints of the application and  endpoints that are meant to be loaded in a cross-site context can be exempted from the policy.
+
+The browser provides information about the context of an HTTP request in a set of `Sec-Fetch-*` headers. This allows the server processing the request to make decisions on whether the request should be accepted or rejected based on the preferred resource isolation policy. Struts provides a default Resource Isolation Policy that rejects cross-origin requests that aren't top level navigations.
+
+```
+Sec-Fetch-Site == 'cross-site' AND (Sec-Fetch-Mode != 'navigate'/'nested-navigate' OR method NOT IN [GET, HEAD])
+```
+
+Refer to [Implementing a Resource Isolation Policy](https://web.dev/fetch-metadata/#implementing-a-resource-isolation-policy) for further information on implementing effective Resource Isolation Policies.
+Fetch Metadata is supported in all major browsers
+
+## Parameters
+
+- `exemptedPaths` - Set of opt out endpoints that are meant to serve cross-site traffic. Paths should either be relative or absolute. No default value for field.
+- `resourceIsolationPolicy` - Instance of Resource Isolation Policy class implementing the logic for the requests filtering. The default value for this field is a new `DefaultIsolationPolicy`.
```

Comment:
This isn't an actual parameter of the interceptor. As a matter of fact, the resource isolation policy cannot be configured at the moment (apart from subclassing the interceptor, perhaps). The [only parameter this interceptor receives from XML config](https://github.com/apache/struts/blob/54b1568a5041f456f98266af7897585b861accc8/core/src/main/java/org/apache/struts2/interceptor/FetchMetadataInterceptor.java#L56) is `exemptedPaths`
```suggestion
```